--- cvsgraph-1.5.0/cvsgraph.conf.orig	2004-08-15 18:36:49.000000000 +0200
+++ cvsgraph-1.5.0/cvsgraph.conf	2004-08-19 13:56:30.000000000 +0200
@@ -86,8 +86,8 @@
 #	CSV/RCS repository can be found
 # cvsmodule <string>
 #
-cvsroot		= "/var/cvs";
-cvsmodule	= "";
+cvsroot		= "/home/services/cvsroot";
+cvsmodule	= "local";
 
 # color_bg <color>
 #	The background color of the image
@@ -137,7 +137,7 @@
 #	Sets the error/warning message font
 msg_color	= "#800000";
 msg_font	= medium;
-msg_ttfont	= "/usr/share/fonts/bitstream-vera/VeraIt.ttf";
+msg_ttfont	= "/usr/share/fonts/TTF/ariali.ttf";
 msg_ttsize	= 11.0;
 
 # parse_logs <boolean>
@@ -163,7 +163,7 @@
 #	will be shown.
 #	Note: tags matched with merge_from/merge_to will still be displayed.
 tag_font	= medium;
-tag_ttfont	= "/usr/share/fonts/bitstream-vera/VeraIt.ttf";
+tag_ttfont	= "/usr/share/fonts/TTF/ariali.ttf";
 tag_ttsize	= 11.0;
 tag_color	= "#007000";
 #tag_ignore	= "(test|alpha)_release";
@@ -171,7 +171,7 @@
 #tag_negate	= false;
 
 rev_font	= giant;
-rev_ttfont	= "/usr/share/fonts/bitstream-vera/Vera.ttf";
+rev_ttfont	= "/usr/share/fonts/TTF/arial.ttf";
 rev_ttsize	= 12.0;
 rev_color	= "#000000";
 rev_bgcolor	= "#f0f0f0";
@@ -184,7 +184,7 @@
 rev_bspace	= 3;
 rev_text	= "%d";	# or "%d\n%a, %s" for author and state too
 rev_text_font	= tiny;
-rev_text_ttfont	= "/usr/share/fonts/bitstream-vera/VeraSe.ttf";
+rev_text_ttfont	= "/usr/share/fonts/TTF/times.ttf";
 rev_text_ttsize	= 9.0;
 rev_text_color	= "#500020";
 rev_maxtags	= 25;
@@ -261,11 +261,11 @@
 #	tag. This option you would normally want to set from the
 #	commandline with the -O option.
 branch_font	= medium;
-branch_ttfont	= "/usr/share/fonts/bitstream-vera/VeraBd.ttf";
+branch_ttfont	= "/usr/share/fonts/TTF/arialbd.ttf";
 branch_ttsize	= 18.0;
 branch_tag_color= "#000080";
 branch_tag_font	= medium;
-branch_tag_ttfont = "/usr/share/fonts/bitstream-vera/VeraBI.ttf";
+branch_tag_ttfont = "/usr/share/fonts/TTF/arialbi.ttf";
 branch_tag_ttsize = 14.0;
 branch_color	= "#0000c0";
 branch_bgcolor	= "#ffffc0";
@@ -295,7 +295,7 @@
 title_x		= 10;
 title_y		= 5;
 title_font	= small;
-title_ttfont	= "/usr/share/fonts/bitstream-vera/VeraSe.ttf";
+title_ttfont	= "/usr/share/fonts/TTF/times.ttf";
 title_ttsize	= 10.0;
 title_align	= left;
 title_color	= "#800000";
